### ISSUE TITLE
Psalm and PHPStan not working

### ISSUE THREAD

**Issue by aimfeld** [lando/lando#2796]
I'm trying to set up Psalm and PHPStan using a remote PHP interpreter via docker. However, both Psalm and PHPStan crash with the following errors:

![image](https://user-images.githubusercontent.com/760330/103881827-41fb9a00-50db-11eb-9795-4b6226f4bf59.png)

I'm using the platform.sh recipe and this is how my remote interpreter setup looks like:

![image](https://user-images.githubusercontent.com/760330/103883323-5b9de100-50dd-11eb-9e1d-ecee12e3d841.png)

![image](https://user-images.githubusercontent.com/760330/103883425-796b4600-50dd-11eb-856c-3fbe37ac492e.png)

There is also a devwithlando/php:7.3-apache-2 container, but I need PHP 7.4, not 7.3:

![image](https://user-images.githubusercontent.com/760330/103883677-d404a200-50dd-11eb-8984-842dfb2691a9.png)

Not sure why the php.ini file is not found (or if this is a problem or not), it actually exists
![image](https://user-images.githubusercontent.com/760330/103882283-de25a100-50db-11eb-90f1-9f2685577b30.png)

```
web@44f956551a8a:/etc/php/7.4-zts/cli$ ls -al /etc/php/7.4-zts/cli/php.ini
lrwxrwxrwx 1 root root 41 Oct  9 05:04 /etc/php/7.4-zts/cli/php.ini -> /run/.configs/etc/php/7.4-zts/cli/php.ini
```

**Comment by aimfeld** [comment]
I tried again and it looks like now there is now also PHP 7.4 devwithlando container available (devwithlando/php:7.4-apache-2). It's still not working though, it seems like psalm.xml is not found even though it exists in the container:
![image](https://user-images.githubusercontent.com/760330/107208019-4c25f680-6a01-11eb-8576-b5b958206890.png)

**Comment by aimfeld** [comment]
Even though this warning is shown, it seems like psalm.xml is working anyway. The `devwithlando/php:7.4-apache-2` seems to be working fine and I now get psalm inspections in the PHPStorm editor and inspection results. I haven't gotten PHPStan working yet, since the PHP cli is now working, I'll close this issue.
